version.json

```diff
@@ -0,0 +1 @@
+{"versionName":"2021.5.28","versionCode":100000039}
```

Comment:
You hat to try quite a lot it seems :astonished: :exploding_head:

Reply:
Why did we start in this project with 100000000 again? :see_no_evil: